src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -910,6 +917,17 @@ def on_demand_apdex_snql_factory(
     )
 
 
+def on_demand_count_web_vitals_snql_factory(
+    aggregate_filter: Function, org_id: int, use_case_id: UseCaseID, alias: Optional[str] = None
+):
+    return _count_if_with_conditions(
+        [
+            _web_vital_equivalence(org_id, TransactionSatisfactionTagValue.SATISFIED.value),
```

Comment:
I don't know the definition of count web vitals but I expect the `measurement_rating` to be in (`meh` `poor` `any` `good`), am I missing something?

Reply:
Ditto needed to remove this, was from wip